src/main/java/frc5124/robot2020/subsystems/Hanger.java

```diff
@@ -16,44 +16,60 @@
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Subsystem;
 import frc5124.robot2020.RobotMap;  
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 
 public class Hanger implements Subsystem {
   private CANSparkMax hangerMotor;
-  private Solenoid brake;
+  private Solenoid noBrake;
```

Comment:
seems odd to use the negative here, would be clearer to read code if brake is on or off rather than the no brake is true or false.